Bottom: bc1991e37b6c613e42c22b06ba1fa2341a65d961
Top:    400723ee2e75545ed34774f6279e1969274b1bd3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:56:11 +0000

Disable small edge EPSILON hack for computing edge normals


---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index a04d8eb..468d966 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -227,7 +227,7 @@ object3d_to_step_body_fragment (step_file *step,
           dir_y = dv->y - ov->y;
           dir_z = dv->z - ov->z;
 
-#if 1
+#if 0
           /* XXX: This avoids the test file step_outline_test.pcb failing to display properly in freecad when coordinates are slightly rounded */
           if (dir_x < EPSILON && -dir_x < EPSILON &&
               dir_y < EPSILON && -dir_y < EPSILON &&
